src/main/java/seedu/address/logic/commands/project/DeleteProjectCommand.java

```diff
@@ -48,10 +49,21 @@ public CommandResult execute(Model model, Ui ui) throws CommandException {
         Project projectToDelete = lastShownList.get(targetIndex.getZeroBased());
         List<Issue> listOfIssuesToDelete = projectToDelete.getIssueList();
         for (Issue i : listOfIssuesToDelete) {
-            UniqueIssueList.remove(i);
+            model.deleteIssue(i);
         }
         model.deleteProject(projectToDelete);
 
+        if (!projectToDelete.getClient().isEmpty()) {
+            projectToDelete.getClient().getProjects().remove(projectToDelete);
+            if (projectToDelete.getClient().getProjects().size() == 0) {
```

Comment:
Can consider using a single telling the project to compute the size of it's client's projects or remove the project as one of its methods

Reply:
Suggestion added